core/primitives/transaction_validity.hpp

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_PRIMITIVES_TRANSACTION_VALIDITY_HPP_
+#define KAGOME_CORE_PRIMITIVES_TRANSACTION_VALIDITY_HPP_
+
+#include <variant>
+#include <vector>
+
+namespace kagome::primitives {
+
+  namespace {
+    /// Priority for a transaction. Additive. Higher is better.
```

Comment:
"Better" is not a good word when speaking about priorities. Maybe, reformulate the statement?

Reply:
It is taken from substrate:) I will leave a link here

Reply:
"Higher means more urgent" ?